docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1315,6 +1315,40 @@ When specifying file names, Teku expects that the files exist.
 
 List of validator public keys used by an external signer (for example, Web3Signer).
 
+### validators-external-signer-slashing-protection-enabled
+
+=== "Syntax"
+
+    ```bash
+    --validators-external-signer-slashing-protection-enabled[=<BOOLEAN>]
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --validators-external-signer-slashing-protection-enabled=false
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    TEKU_VALIDATORS_EXTERNAL_SIGNER_SLASHING_PROTECTION_ENABLED=false
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    validators-external-signer-slashing-protection-enabled: false
+    ```
+
+Specify whether to use Teku's built-in [slashing protection] when using an external signer such as
+[Web3Signer]. Defaults to `true`.
```

Comment:
Can we add a big warning that users should be absolutely certain the external signer has slashing protection enabled before disabling this protection. Otherwise their validator may get slashed.

Reply:
Done